### DISCUSSION TITLE
Bubble munu in read-only editor

### DISCUSSION THREAD

**Discussion by steFanBzh** [ueberdosis/tiptap#3127] [General Questions]
Is it possible to trigger the bubble menu on selection update in read only mode?

**Comment by sereneinserenade** [comment]
yes, use [`shouldShow`](https://tiptap.dev/api/extensions/bubble-menu#should-show) example: https://github.com/sereneinserenade/tiptap-comment-extension/blob/main/src/components/Tiptap.vue#L201
